src/Styleguide/Components/EntityHeader.tsx

```diff
@@ -31,9 +31,11 @@ export class EntityHeader extends Component<EntityHeaderProps> {
             </Serif>
 
             <Flex>
-              <Sans size="2" color="black60">
-                {meta}
-              </Sans>
+              {!!meta && (
```

Comment:
@damassi - curious why in the EntityHeader some properties are defined as optional, like `initials?: string` where meta is defined as `meta: string | null`. Why not `meta?: string`?

Reply:
This is carryover from your previous feedback here: https://github.com/artsy/reaction/pull/1275#discussion_r219944642. I can update though